tests/cases/standalone/common/alter/alter_table.result

```diff
@@ -165,6 +165,16 @@ FROM
 | loc_1 | loc_2 | loc_3 | 2   | job1 | 1970-01-01T00:00:00 | 1.0 |
 +-------+-------+-------+-----+------+---------------------+-----+
 
+-- issue #3235 can not alter when table has column with tyep interval
+create table t3(val interval, ts timestamp time index);
```

Comment:
As mentioned in https://github.com/GreptimeTeam/greptimedb/issues/3235#issuecomment-2606540702. We can return an error if users want to create a table with interval type. It won't break users as they can't use tables with interval types now.

They can use integers or floating points to store elapsed time. We usually use interval types when writing a SELECT SQL.

Reply:
Thanks for the info will change the error return way in `create table` side

Reply:
@evenyag done and fix the break tests